File: tests/conformance/fixtures/go/err-cycle.go
package main

import sykli "github.com/false-systems/sykli/sdk/go"

func main() {
	p := sykli.New()

	p.Task("A").Run("echo A").After("B")
	p.Task("B").Run("echo B").After("A")

	p.Emit()
}

File: tests/conformance/fixtures/elixir/err-cycle.exs
use Sykli

pipeline do
  task "A" do
    run "echo A"
    after_ ["B"]
  end
  task "B" do
    run "echo B"
    after_ ["A"]
  end
end
|> Sykli.Emitter.validate!()
|> Sykli.Emitter.to_json()
|> IO.puts()
